app/models/account.rb

```diff
@@ -37,4 +39,8 @@ class Account < ActiveRecord::Base
   def downcase_subdomain
     self.subdomain = subdomain.try(:downcase)
   end
+
+  def single_tenant_mode?
+    Hours.single_tenant_mode?
+  end
```

Comment:
Right, missed that you're using this in the validation here, this is fine then! :)